src/app/api/project/[id]/route.ts

```
@@ -0,0 +1,171 @@
+import { NextRequest } from "next/server";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)